ecosystem/indexer/src/models/events.rs

```diff
@@ -3,7 +3,8 @@
 #![allow(clippy::extra_unused_lifetimes)]
 use crate::{models::transactions::Transaction, schema::events};
 use aptos_rest_client::aptos_api_types::Event as APIEvent;
-use serde::Serialize;
+use diesel::{data_types::PgNumeric};
+use serde::{Serialize, Serializer};
 
 #[derive(Associations, Debug, Identifiable, Insertable, Queryable, Serialize)]
```

Comment:
Another issue is this "Insertable". this trait is not implemented for PgNumeric. Thus it cannot be derived. We may need a custom implementation of the insertable trait.  This is challenging based on my current understanding of RUST. The pgNumeric is not local type and also Insertable is not a local trait as well.